src/main/java/de/florianmichael/viaforge/mixin/impl/MixinGuiMultiplayer.java

```diff
@@ -35,6 +36,8 @@ public class MixinGuiMultiplayer extends GuiScreen {
 
     @Inject(method = "initGui", at = @At("RETURN"))
     public void hookViaForgeButton(CallbackInfo ci) {
+        ViaForgeCommon.init(ViaForge189.PLATFORM);
```

Comment:
This should have a noted why it's needed but I would general prefer if we could move loading to an Forge event just like on the main branch?